Sprint-3/revise/implement/password-validator.js

```diff
@@ -0,0 +1,12 @@
+function isValidPassword(password, passwords) {
```

Comment:
Your function only checks for one of conditions:
```
To be valid, a password must:
- Have at least 5 characters.
- Have at least one English uppercase letter (A-Z)
- Have at least one English lowercase letter (a-z)
- Have at least one number (0-9)
- Have at least one of the following non-alphanumeric symbols: ("!", "#", "$", "%", ".", "*", "&")
- Must not be any previous password in the passwords array. 
```

You could also include more test cases in the test script.